src/modules/launcher/Wox.Core/Plugin/PluginManager.cs

```diff
@@ -185,6 +186,23 @@ public static List<Result> QueryForPlugin(PluginPair pair, Query query, bool del
             }
         }
 
+        private static List<Result> UpdateResultWithActionKeyword(List<Result> results, Query query)
+        {
+            foreach (Result result in results)
+            {
+                if (!string.IsNullOrEmpty(result.QueryTextDisplay))
```

Comment:
I think !string.IsNullOrWhiteSpace() would be better in this case.

Reply:
A whitespace would only be present if we had explicitly set it from a plugin. I think it is better handled at the plugin end.